mlflow/server/js/src/common/utils/Utils.js

```diff
@@ -191,7 +191,7 @@ class Utils {
    * @param endTime in milliseconds
    */
   static getDuration(startTime, endTime) {
-    return startTime && endTime ? this.formatDuration(endTime - startTime) : null;
+    return startTime && endTime ? this.formatDuration(endTime - startTime) : '';
```

Comment:
I think returning `null` here is semantically clearer than returning the empty string. Can we maintain the `null` return type and handle the mapping from `null` to empty string in the CSV export routine?

Reply:
Agreed. Reverted back to returning `null` here. 

For consistency, should this PR refactor the other getter functions in [Utils](https://github.com/mlflow/mlflow/blob/master/mlflow/server/js/src/common/utils/Utils.js) (getRunName, getSourceName etc.) to return `null` instead of empty strings? Let me know!